HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,88 @@
+# Trace Context HTTP Header Format
+
+Date: 31/03/2017
+
+A trace context header is used to pass trace context information across systems
+for a HTTP request. Our goal is to share this with the community so that various
+tracing and diagnostics products can operate together, and so that services can
+pass context through them, even if they're not being traced (useful for load
+balancers, etc.)
+
+# Format
+
+## Header name
+
+`Trace-Context`
```

Comment:
`Trace-Context` can be mistaken with something carrying the baggage or context properties. This is `id`. We decided to use `Request-Id` for correlation protocol in .NET. If we will be able to converge on format - it may be a good one to use.

`Request-Id` is quite descriptive for the header purpose I think.